src/groupByKey.ts

```diff
@@ -1,7 +1,16 @@
-type GroupsMap<T> = {
-  [key: string]: T[];
-};
+interface Item {
+  [key: string]: any;
```

Comment:
By using `any` you miss the point of typescript) It should be of generic type

`  [key: string]: T[];`

Reply:
@dmtrhrytsak  I was just so fed up with the complexity of some typescript stuff that I learned recently, that I wanted to simplify things a bit) But you are right, I better change it)

Reply:
No, that's okay, I understand you) You deserve resting not this whole typescript thing! :D